src/features/devtools/components/DevToolsLogs/DevToolsLogs.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { Button, TextField } from '@digdir/design-system-react';
+import {
+  DownloadIcon,
+  ExclamationmarkTriangleFillIcon,
+  InformationSquareFillIcon,
+  TrashIcon,
+  XMarkOctagonFillIcon,
+} from '@navikt/aksel-icons';
+
+import classes from 'src/features/devtools/components/DevToolsLogs/DevToolsLogs.module.css';
+import { DevToolsActions } from 'src/features/devtools/data/devToolsSlice';
+import { useAppSelector } from 'src/hooks/useAppSelector';
+
+const colorMap = {
+  error: 'red',
+  warn: 'darkorange',
+  info: 'black',
+};
+
+export const DevToolsLogs = () => {
+  const logs = useAppSelector((state) => state.devTools.logs);
+  const [filter, setFilter] = useState('');
+  const [showLevels, setShowLevels] = useState({ error: true, warn: true, info: true });
+
+  const dispatch = useDispatch();
+  const clearLogs = () => dispatch(DevToolsActions.logsClear());
+  const toggleShow = (level: string) => setShowLevels({ ...showLevels, [level]: !showLevels[level] });
+  const saveLogs = () => {
+    const data = logs.map((log) => `${log.index + 1}. - ${log.level.toUpperCase()}: ${log.message}`).join('\n\n');
+    const blob = new Blob([data], { type: 'text/plain;charset=utf-8' });
+    const url = URL.createObjectURL(blob);
+    const a = document.createElement('a');
+    a.setAttribute('href', url);
+    a.setAttribute('download', `${window.org}-${window.app}-${new Date().toISOString()}-logs.txt`);
+    a.click();
```

Comment:
🤩

Reply:
Yeah seems extremely hacky, but this really is how people are doing it 😅